### ISSUE TITLE
cli no exitcode when compile

### ISSUE THREAD

**Issue by janneman001** [bytenode/bytenode#208]
bytenode only sends a console error when compile errors occur, no exitcode.
this way ci-jobs do not stop and errors can pass unnoticed.

**Comment by OsamaAbbas** [comment]
Yes, that'a a known bug that I will fix in the upcoming version. I was just very busy in the past few months.

Check out the [workaround here](https://github.com/bytenode/bytenode/issues/204) and let me know if it solves your issue.

**Comment by janneman001** [comment]
Thanks, also for your tool. I use it for ip.
